internal/command/workspace_delete.go

```diff
@@ -210,7 +210,7 @@ Usage: terraform [global options] workspace delete [OPTIONS] NAME
 
 Options:
 
-  -force             Remove even a non-empty workspace.
+  -force             Remove a workspace even if it is managing resources. Terraform can no longer track or manage the workspace's infrastructure. 
```

Comment:
We need to wrap this help text manually, as Terraform doesn't perform word wrapping for us. Without word wrap it breaks awkwardly mid-word at the terminal edge:

<img width="857" alt="image" src="https://user-images.githubusercontent.com/68917/204024903-103dd503-f5b0-4b62-abfc-ed182eb2b758.png">

I think we manually wrap at 76 characters by convention.